arch/sim/src/sim/up_ajoystick.c

```diff
@@ -87,6 +87,7 @@ static const struct ajoy_lowerhalf_s g_ajoylower =
 
 /* Driver state data */
 
+staitc int g_eventloop;
```

Comment:
static is mispelled.  This could not have compiled!

Reply:
We use touchscreen config, can't find this error locally. I will update the patch.